### ISSUE TITLE
Stream update allowing undeploy/re-deploy selected applications

### ISSUE THREAD

**Issue by ilayaperumalg** [spring-cloud/spring-cloud-dataflow#4061]
As a user, I'd like to have an option to selectively undeploy/re-deploy selected applications inside a Stream.

This will help me re-deploy my app to refresh on some external dependencies.

**Comment by sabbyanandan** [comment]
Perhaps the option in the stream's force update via `--app-names` and `--force` flag may already address what you're looking for. WDYT?

**Comment by ilayaperumalg** [comment]
Yes, I intended to have them created in the SCDF UI but created here. Moving there.
